tedana/config/img_table.json

```diff
@@ -5,11 +5,11 @@
     },
     "t2star map": {
         "orig": "t2sv",
-        "bidsv1.5.0": "T2starmap"
+        "bidsv1.5.0": "desc-T2starmap"
```

Comment:
T2starmap and S0map are full-on suffixes, rather than values associated with an entity (e.g., `desc`).

Reply:
So I should actually revert this change?